example/viam_example_app/lib/screens/motor.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_platform_widgets/flutter_platform_widgets.dart';
+import 'package:viam_sdk/viam_sdk.dart';
+
+class MotorScreen extends StatefulWidget {
+  final Motor motor;
+  final ResourceName resourceName;
+
+  const MotorScreen({Key? key, required this.motor, required this.resourceName}) : super(key: key);
+
+  @override
+  State<MotorScreen> createState() {
+    return _MotorScreenState();
+  }
+}
+
+class _MotorScreenState extends State<MotorScreen> {
+  double power = 0;
+  bool isOn = false;
+
+  void moveMotor(double powerPct) {
+    widget.motor.setPower(powerPct / 100);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return PlatformScaffold(
+      appBar: PlatformAppBar(
+        title: Text(widget.resourceName.name.toUpperCase()),
+      ),
+      iosContentPadding: true,
+      body: Center(
+        child: Column(
+          children: [
+            const SizedBox(height: 16),
+            PlatformText(
+              '${widget.resourceName.namespace}:${widget.resourceName.type}:${widget.resourceName.subtype}/${widget.resourceName.name}',
```

Comment:
Recurrence of the empty line here makes me think this is just a typical dart thing, but figured wanted to flag here as well as above just in case :)

Reply:
yes, similar as before.

The PlatformText widget in this case automatically formatted to take up multiple lines because of the line lengths in the separate fields, and how indented in it is in the tree. (This is a setting you can dial in in the flutter/dart linter).

Essentially what is going on here is:
```
PlatformText(String 'text', Style TextSty.(params),),
```

but it gets broken out to

```
PlatformText(
   'text',
   Style: TextStyle(params...),
),
```

if the arguments for `TextStyle(...)` were more verbose then that would potentially get broken out too.

It doesn't affect how the code gets executed, this is just an attempt to standardize line length / readability automatically.

Reply:
@stuqdog I aligned the settings between the outer project and the nested example project in this PR https://github.com/viamrobotics/viam-flutter-sdk/pull/16